tests/lib/kubernetes/kubernetes_base.py

```diff
@@ -82,10 +83,80 @@ def kubeconfig(self):
     def kubectl_exec(self):
         return self._kubectl_exec
 
+    def gather_logs(self, dest_dir):
```

Comment:
nit: I would remove `dest_dir` and directly access the settings variable.

Reply:
If you are in a playground mode or debugger etc, this may still be useful to grab everything to a directory that you give it.